### DISCUSSION TITLE
Always encounter "AssertionError: You do not have CLIP state dict!"

### DISCUSSION THREAD

**Discussion by shiboyang** [lllyasviel/stable-diffusion-webui-forge#1923] [Q&A]
Use some of ckpt to generate the image will raise "You do not have CLIP state dict!"
Has anyone encounted this situation?
the sdxl-base model can work, however the sdxl-refine model sometime not work

**Comment by Jonseed** [comment]
I'm also seeing this error on the SDXL refiner. Not sure how to fix it.

**Comment by dallas4u** [comment]
Make sure your text encoders (clip_l and t5-xxl) are in the **\models\text_encoder** folder
Make sure your VAE are in the **\models\VAE** folder
Then, start forge and test

**Reply by Jonseed** [reply]
yes, they are in the correct locations, and I still get the error.

**Comment by usigg3** [comment]
At least with Flux you need to select them in the "VAE / Text Encoder" window, you can select several there.
